types/artifact.go

```diff
@@ -52,6 +52,14 @@ type Application struct {
 	Libraries []LibraryInfo
 }
 
+type CustomResource struct {
+	FilePath string
+	Type     string
```

Comment:
nit: be aligned with other types

```suggestion
	Type     string
	FilePath string
```